docs/openapi.yaml

```diff
@@ -1939,9 +1979,6 @@ paths:
                 file:
                   type: string
                   format: binary
-            encoding:
```

Comment:
Why was this removed? Probably related to the above changes

Reply:
Changed and pushed